src/api.js

```diff
@@ -57,7 +57,13 @@ export async function postReview(review, verified = false) {
 }
 
 export async function getScreenings(query = "") {
-  const res = await fetch(APIData + "/screenings?populate=movie");
+  const resPag = await fetch(APIData + "/screenings?populate=movie");
+  const payPag = await resPag.json();
+  console.log(payPag.meta.pagination.total);
+  const res = await fetch(
+    APIData +
+      `/screenings?populate=movie&pagination[pageSize]=${payPag.meta.pagination.total}`
```

Comment:
Great! You would think not including query string for pagination would respond with all screenings. Seems strange having to do 2 requests/fetch to get all of them. Nice solution! :)